module_analysis/views/menu.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- 
+Copyright (C) 2019-Today: GRAP (<http://www.grap.coop/>)
+@author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+ -->
+
+<odoo>
+
+    <menuitem id="menu_module_analysis"
+        name="Module Analysis"
```

Comment:
To be more mobile friendly it may be nice to use shorter name.
Here we are in module menu then no confusion

```suggestion
        name="Analysis"
```